* Fix build with cmake>=4: this package does not seem to use CMP0026 and CMP0042
* Avoid installing garbage

--- CMakeLists.txt~	2016-05-07 21:07:04.000000000 +0200
+++ CMakeLists.txt	2016-06-16 11:52:34.663429267 +0200
@@ -156,13 +156,6 @@
 #
 ################################################################################
 
-# Suppressing some stupid warnings from CMake 3.0
-IF (POLICY CMP0026)
-	CMAKE_POLICY(SET CMP0026 OLD)
-ENDIF( )
-IF (POLICY CMP0042)
-	CMAKE_POLICY(SET CMP0042 OLD)
-ENDIF( )
 
 #
 # Compiler options are in separate file
@@ -242,6 +236,7 @@
 	DESTINATION
 		${INSTALL_INCLUDE_DIR}
 	FILES_MATCHING PATTERN "*.hpp"
+	PATTERN "*Make*" EXCLUDE
 )
 INSTALL(
 	DIRECTORY
@@ -249,17 +244,9 @@
 	DESTINATION
 		${INSTALL_INCLUDE_DIR}
 	FILES_MATCHING PATTERN "*.ipp"
+	PATTERN "*Make*" EXCLUDE
 )
 
-#
-# Install Eigen3
-#
-INSTALL(
-	DIRECTORY
-		external_packages/eigen3
-	DESTINATION
-		${INSTALL_INCLUDE_DIR}/external_packages
-)
 
 #
 # Install qpOASES embedded
